static/js/admin.js

```diff
@@ -19,6 +19,8 @@ var admin_settings_label = {
     realm_name_changes_disabled: i18n.t("Prevent users from changing their name"),
     realm_email_changes_disabled: i18n.t("Prevent users from changing their email address"),
     realm_avatar_changes_disabled: i18n.t("Prevent users from changing their avatar"),
+    realm_waiting_period_applies_to_create_stream: i18n.t("Creating streams"),
+    realm_waiting_period_applies_to_invite_to_stream: i18n.t("Inviting other members to streams"),
```

Comment:
Let's make this "Adding users to streams", since we don't really have an "Inivtation" concept yet, and also this applies more broadly than just members.